src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -247,6 +247,7 @@ public static void AddScale(float a, ReadOnlySpan<float> src, Span<float> dst, i
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why are we only marking some methods with `[MethodImpl(MethodImplOptions.AggressiveInlining)]` and not all?

Similarly, why are we checking `if VectorizationMakesSense` for only some methods, but not all?